Model/AdyenThreeDS2Process.php

```diff
@@ -90,6 +103,7 @@ public function initiate($payload)
             // unset payment data from additional information
             $payment->unsAdditionalInformation("threeDS2PaymentData");
         } else {
+            $this->adyenLogger->addAdyenDebug("3D secure 2.0 failed, payment data not found");
```

Comment:
```suggestion
            $this->adyenLogger->error("3D secure 2.0 failed, payment data not found");
```
should we use error?